tests/test_zmq.py

```diff
@@ -0,0 +1,104 @@
+from swsscommon import swsscommon
+
+from dash_api.appliance_pb2 import *
+from dash_api.vnet_pb2 import *
+from dash_api.eni_pb2 import *
+from dash_api.route_pb2 import *
+from dash_api.route_rule_pb2 import *
+from dash_api.vnet_mapping_pb2 import *
+from dash_api.route_type_pb2 import *
+from dash_api.types_pb2 import *
+
+import typing
+import time
+import binascii
+import uuid
+import ipaddress
+import sys
+import socket
+import logging
+import pytest
+
+logging.basicConfig(level=logging.INFO)
+zmq_logger = logging.getLogger(__name__)
+
+DVS_ENV = ["HWSKU=DPU-2P"]
+NUM_PORTS = 2
+
+class Table(object):
+    def __init__(self, database, table_name: str):
+        self.table_name = table_name
+        self.table = swsscommon.Table(database.db_connection, self.table_name)
+
+    def __getitem__(self, key: str):
+        exists, result = self.table.get(str(key))
+        if not exists:
+            return None
+        else:
+            return dict(result)
+
+    def get_keys(self):
+        return self.table.getKeys()
+
+    def get_newly_created_oid(self, old_oids):
+        new_oids = self.asic_db.wait_for_n_keys(table, len(old_oids) + 1)
+        oid = [ids for ids in new_oids if ids not in old_oids]
+        return oid[0]
+
+class DashZmq(object):
+    def __init__(self, dvs):
+        self.dvs = dvs
+        self.asic_direction_lookup_table = Table(
+            self.dvs.get_asic_db(), "ASIC_STATE:SAI_OBJECT_TYPE_DIRECTION_LOOKUP_ENTRY")
+        self.asic_vip_table = Table(
+            self.dvs.get_asic_db(), "ASIC_STATE:SAI_OBJECT_TYPE_VIP_ENTRY")
+
+class TestZmqDash(object):
+    @pytest.fixture(scope="class")
+    def enable_orchagent_zmq(self, dvs):
+        # change orchagent to use ZMQ
+        dvs.runcmd("cp /usr/bin/orchagent.sh /usr/bin/orchagent.sh_zmq_ut_backup")
```

Comment:
Because ZMQ version issue, we can't send ZMQ message on host, so need upload and run script on SONiC KVM to send ZMQ message.

Reply:
What is "ZMQ version issue"?

Reply:
The test_zmq.py are running on ubuntu VM, and the ubuntu version swsscommon lib crash when send ZMQ message, seems the ZMQ lib have compatibility issue.